radioapp/ui/ui_manager.py

```diff
@@ -1,45 +1,47 @@
-import tkinter
-from tkinter import ttk
 from .home import UIHome
-from .settings import UISettings
+import tkinter as tk
+
+
+class UIManager:
+    """Manages UI Screens and Settings."""
 
-class UI():
-    """Holds the main window."""
-    
     def __init__(self):
-         """
-         Initializes window and frame.
-         
-         Set defaults for the tkinter instance.
-         """
-        
-        self.window = tkinter.Tk()
-        self.window.columnconfigure(0, weight=1, uniform=1)
-        self.window.rowconfigure(0, weight=1, uniform=1)
+        """Creates a window, and manages different screens and settings."""
+
+        self.current_slider_number = 0
+        self.current_screen = None
+
+        self.initialize_window()
+
+        self.switch_screen(self.display_home, 0)
+
+    def display_home(self):
+        """Displays the Home screen."""
+
+        self.current_screen = UIHome(self.window, self)
+        self.current_screen.pack_screen()
+
+    def initialize_window(self):
+        """Initializes the main window."""
+
+        self.window = tk.Tk()
         self.window.geometry("160x128")
-        self.window.resizable(0,0)
+        self.window.configure(background="#3399ff")
+        self.window.resizable(0, 0)
         self.window.pack_propagate(0)
-        
-        #Set defaults for the tkinter instance.
-        self.style = ttk.Style(self.window)
-        self.style.configure('TFrame', background='blue')
-        self.window.option_add("*Background", "blue")
-        self.window.option_add("*Foreground", "white")
-
-        self.start()
-        
-    def start(self):
-        self.go_home()
-        self.window.mainloop()
-
-    def go_home(self):
-        self.currentPage = UIHome(self.window, self.go_settings, None, None)
 
-    def go_settings(self):
-        self.currentPage = UISettings(self.window, self.go_home)
+    def switch_screen(self, screen_class, action_number):
+        """Switches between Screen."""
 
+        if action_number == 1:
+            self.current_slider_number = self.current_screen.current_slider_number
```

Comment:
Did this accidentally come over from the backlight screen?

Reply:
It comes from Backlight screen since I have already integrated Backlight functionality with the home screen. I have removed the Backlight screen for this pull request. It will be obvious when I upload the Backlight functionality part.

Reply:
See above comment